contracts/InterchainTokenService.sol

```diff
@@ -176,6 +176,16 @@ contract InterchainTokenService is
         _;
     }
 
+    /**
+     * @notice This modifier is used to ensure that only an operator or the owner can call a function
+     */
+    modifier onlyOperatorOrOwner() {
+        address sender = msg.sender;
+        if (owner() != sender && !hasRole(sender, uint8(Roles.OPERATOR))) revert NotOperatorOrOwner(sender);
```

Comment:
```suggestion
        address sender = msg.sender;

        if (!hasRole(sender, uint8(Roles.OPERATOR)) && sender != owner()) revert NotOperatorOrOwner(sender);
```
check for operator first since it's more likely, so less gas cost